sig-cli/charter.md

```diff
@@ -5,25 +5,81 @@ the Roles and Organization Management outlined in [sig-governance].
 
 ## Scope
 
-The Command Line Interface SIG (SIG CLI) is responsible for kubectl and
-related tools. This group focuses on general purpose command line tools and
-libraries to interface with Kubernetes API's.
-
 ### In scope
 
-SIG CLI [README]
-
-#### Code, Binaries and Services
-
-SIG CLI code include general purpose command line tools and binaries for working
-with Kubernetes API's. Examples of these binaries include: [kubectl and kustomize].
+The Command Line Interface SIG (SIG CLI) is responsible for kubectl and
+related tools that are developed by the SIG.  See kubectl
+[design principles](design-principles.md) for the focus of kubectl functionality.
+
+This group focuses on command line tooling for working with Kubernetes APIs and Resource
+Config.  This includes both generalized tooling for working with Resources, Resource Config
```

Comment:
> This group focuses on command line tooling for working with Kubernetes APIs and Resource
with Kubernetes API's.

Should this use the same language as [the in work Kubernetes scope document](https://github.com/kubernetes/community/pull/3180) that notes the CLI is basic and a reference implementation.